pkg/services/ngalert/api/api_ruler.go

```diff
@@ -368,7 +342,7 @@ func (srv RulerSrv) RoutePostNameRulesConfig(c *models.ReqContext, ruleGroupConf
 
 // updateAlertRulesInGroup calculates changes (rules to add,update,delete), verifies that the user is authorized to do the calculated changes and updates database.
 // All operations are performed in a single transaction
-//nolint: gocyclo
+// nolint: gocyclo
```

Comment:
compiler directive - scoot against the //

Reply:
```suggestion
//nolint: gocyclo
```